Dockerfile

```diff
@@ -17,7 +25,7 @@ USER root
 RUN mkdir var/logs && chown www-data:www-data var/logs
 
 USER www-data
-CMD ["venv/bin/python"]
+CMD [".venv/bin/python"]
```

Comment:
This is the remaining issue. You appear not to be able to control the name (https://github.com/pypa/pipenv/issues/1184), and it's different to what we use now. Going to see if a symlink works.